mbuild/bond_graph.py

```diff
@@ -98,6 +98,45 @@ def subgraph(self, nodes):
                     new_graph.add_edge(node, neighbor)
         return new_graph
 
+    """
```

Comment:
I would put this at the top and just mentioned the functions by name. Makes the code a bit cleaner to read.